src/components/spaces/organisms/organisms-container.tsx

```diff
@@ -57,14 +58,27 @@ export class OrganismsContainer extends BaseComponent<IProps, IState> {
     const { rowIndex } = this.props;
     const organismsRow = organisms.rows[rowIndex];
 
-    const availableZoomLevels: ZoomLevelType[] = ["organism", "cell", "protein"];
-    const maxZoom = availableZoomLevels.length - 1;
+    const availableZoomLevels: {[key in ZoomLevelType]: number} = {
+      organism: 0,
+      cell: 1,
+      receptor: 2,
+      nucleus: 2
```

Comment:
should this be 3?

Reply:
Actually, that was the reason I switched it from an array to an object. Receptor and nucleus are at the same zoom level, so when you zoom out from either one you should end up at `1`.

Reply:
that makes more sense - ok, carry on :)